codecov/settings_base.py

```diff
@@ -549,3 +549,10 @@
 
 SHELTER_PUBSUB_PROJECT_ID = get_config("setup", "shelter", "pubsub_project_id")
 SHELTER_PUBSUB_SYNC_REPO_TOPIC_ID = get_config("setup", "shelter", "sync_repo_topic_id")
+
+# Allows to do migrations from another module
+MIGRATION_MODULES = {
+    "codecov_auth": "shared.django_apps.codecov_auth.migrations",
+    "core": "shared.django_apps.core.migrations",
+    "reports": "shared.django_apps.reports.migrations",
+}
```

Comment:
not necessary for legacy_migrations?

Reply:
Good catch, I left it out weeks back when doing this and forgot to put it in. 

In theory, we shouldn't add any new migration here, but it's the correct thing to do. I'll add it in